### PR TITLE
Feat - Added Tests for Helmet Middleware

### PR DESCRIPTION
**Pull request by UtkuErdemir**
@aaydin-tr could you check please?

### CODE REVIEW

**Comment by aaydin-tr** [comment]
Can you seperate middleware tests from different file like `ikari_middlewares.test.ts`

File: __test__/ikari.test.ts [hunk 1]
```diff
@@ -4839,9 +4840,169 @@ describe("Request Cycle", () => {
   });
 });
 
+describe("tests Helmet", () => {
```
Comment (aaydin-tr) [inline __test__/ikari.test.ts:4843]: remove `tests` just `describe("Helmet"` is fine

**Comment by UtkuErdemir** [comment]
> Can you seperate middleware tests and TODO logs from different file like `ikari_middlewares.test.ts`

I have splitted some of tests, if it makes sense let me inform i will split some of them like this because tests file too long.

**Comment by aaydin-tr** [comment]
> > Can you seperate middleware tests and TODO logs from different file like `ikari_middlewares.test.ts`
> 
> I have splitted some of tests, if it makes sense let me inform i will split some of them like this because tests file too long.

I think it's fine the way it is, we just need to separate the tests of the middlewares.

**Review by aaydin-tr — APPROVED** [review]

**Pull request merged**